CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ else()
   else()
     set(WARNING_FLAGS -Werror -Wall -Wno-sign-compare -Wno-unused)
   endif()
-  set(MANIFOLD_FLAGS ${MANIFOLD_FLAGS} ${WARNING_FLAGS} -O3)
```

Comment:
Removing this lets release and debug mode work by default - debug mode actually works great now that we're not on CUDA.

Reply:
Sounds like something caused by `-O3`? Perhaps we got some undefined behavior... I should look into this later.

Reply:
No, the undefined behavior is independent of the amount of optimization.

Reply:
Ah, so removing this line does not make it work, just allows you to compile in debug mode?

Reply:
exactly - and actually I just solved the problem. Turns out I had some extra unused verts that weren't being used, but were being processed and contained undefined stuff. Address sanitizer is great, but even though it doesn't slow down running a debug build that much, it makes the actual debugger unusably slow. Oh well, went back to print statements and figured it out.